variables.tf

```
@@ -19,6 +19,18 @@ variable "sku" {
   default     = "Basic"
 }
 
+variable "system_assigned_identity_enabled" {
+  description = "Should the system-assigned identity be enabled for this Web App?"
```

Correct resource type:

```suggestion
  description = "Should the system-assigned identity be enabled for this Service Bus namespace?"
```